graphics/vtk/Portfile

```diff
@@ -100,7 +102,7 @@ variant qt5 description {Add Qt5 support.} {
 }
 
 # Supported pythons
-set python_versions {27 36 37 38 39 310 311}
+set python_versions {36 37 38 39 310 311}
```

Comment:
did you verify that the PY27 subport doesn't have dependents anymore (of that applies to the PY36 subport as well, feel free to  remove that one as well).

Reply:
> did you verify that the PY27 subport doesn't have dependents anymore

No, that did not occur to me.  I will restore PY27.

Reply:
You should check whether that's the case and if not, by all means remove the PY27 subport

Reply:
CI has now accepted the PY27 subport on default builds.  Is it okay with you if I leave in the PY27 subport?  If so, please mark this review thread as resolved.  Thanks.

Tasks remaining:  Squash commits; re-lint; clean up commit message.